include/glow/Graph/Graph.h

```diff
@@ -780,7 +780,7 @@ class Function final : public Named {
   /// is multiplied by weights[i]. len(weights) must be equal to len(indices).
   FusedRowwiseQuantizedSparseLengthsWeightedSumNode *
   createFusedRowwiseQuantizedSparseLengthsWeightedSum(llvm::StringRef name,
-                                                      Tensor &data,
+                                                      NodeValue data,
```

Comment:
Why'd you swap this parameter with the method below?  Is there some subtlety I'm missing?